care/emr/api/otp_viewsets/login.py

```
@@ -48,41 +85,17 @@ class OTPLoginView(EMRBaseViewSet):
-            random_otp = rand_pass(settings.OTP_LENGTH)
-        else:
-            random_otp = "45612"
-
-        otp_obj = PatientMobileOTP(phone_number=data.phone_number, otp=random_otp)
-        otp_obj.save()
+        data = OTPRequestBaseSpec(**request.data)
+        try:
+            send_otp(data.phone_number, otp_type=LoginOTP())
+        except ValueError as e:
+            raise ValidationError({"phone_number": str(e)}) from e
+        except Exception as e:
```

Ruff (0.15.13)

[warning] Do not catch blind exception: `Exception`

(BLE001)